products/angularjs.md

```diff
@@ -2,16 +2,18 @@
 title: AngularJS
 category: framework
 tags: google javascript-runtime
-iconSlug: angular
 permalink: /angularjs
 alternate_urls:
 -   /angular-js
 versionCommand: npm list angular
 releasePolicyLink: https://docs.angularjs.org/misc/version-support-status
 changelogTemplate: https://github.com/angular/angular.js/blob/v__LATEST__/CHANGELOG.md
-eolColumn: Support
+
 releaseColumn: true
 releaseDateColumn: true
+eolColumn: Support
+
+extendedSupportColumn: Extended Long Term Support
```

Comment:
These changes are to comply the [Style Guide](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide)